localization/english/gui/wc_magic_lifestyles_window_character_lifestyle_l_english.yml

```diff
@@ -4,82 +4,88 @@
- wc_elemental_earth_magic_next_trait_level_is_1:0 "required to get [GetTrait('user_of_elemental_earth_magic_1').GetName( GetPlayer )]: [EmptyScope.ScriptValue('wc_perks_needed_for_level_1_magic_trait_value')|0]"
- wc_elemental_earth_magic_next_trait_level_is_2:0 "required to get [GetTrait('user_of_elemental_earth_magic_2').GetName( GetPlayer )]: [EmptyScope.ScriptValue('wc_perks_needed_for_level_2_magic_trait_value')|0]"
- wc_elemental_earth_magic_next_trait_level_is_3:0 "required to get [GetTrait('user_of_elemental_earth_magic_3').GetName( GetPlayer )]: [EmptyScope.ScriptValue('wc_perks_needed_for_level_3_magic_trait_value')|0]"
- wc_elemental_earth_magic_next_trait_level_is_4:0 "required to get [GetTrait('user_of_elemental_earth_magic_4').GetName( GetPlayer )]: [EmptyScope.ScriptValue('wc_perks_needed_for_level_4_magic_trait_value')|0]"
- wc_elemental_earth_magic_next_trait_level_is_5:0 "required to get [GetTrait('user_of_elemental_earth_magic_5').GetName( GetPlayer )]: [EmptyScope.ScriptValue('wc_perks_needed_for_level_5_magic_trait_value')|0]"
+ WC_LIFESTYLE_WINDOW_PERKS_NEEDED_FOR_NEXT_LIFESTYLE_TRAIT_ELEMENTAL_EARTH_MAGIC:0 "Unlocked [Lifestyle.GetNameNoTooltip] [perks|E] [GetPlayer.Custom('GetWCMagicalLifestyleElementalEarthMagicNextTraitForPerks')]: [Character.GetPerkPointsUsed( Lifestyle.Self )]/[GetPlayer.Custom('GetWCMagicalLifestyleElementalEarthHowMuchPerksForNextTrait')]"
+ wc_elemental_earth_magic_next_trait_level_is_1:0 "required to get [GetTrait('user_of_elemental_earth_magic_1').GetName( GetPlayer )]"
+ wc_elemental_earth_magic_next_trait_level_is_2:0 "required to get [GetTrait('user_of_elemental_earth_magic_2').GetName( GetPlayer )]"
+ wc_elemental_earth_magic_next_trait_level_is_3:0 "required to get [GetTrait('user_of_elemental_earth_magic_3').GetName( GetPlayer )]"
+ wc_elemental_earth_magic_next_trait_level_is_4:0 "required to get [GetTrait('user_of_elemental_earth_magic_4').GetName( GetPlayer )]"
+ wc_elemental_earth_magic_next_trait_level_is_5:0 "required to get [GetTrait('user_of_elemental_earth_magic_5').GetName( GetPlayer )]"
  wc_elemental_earth_magic_next_trait_level_is_last:0 "make [GetPlayer.GetFirstName] a [GetTrait('user_of_elemental_earth_magic_5').GetName( GetPlayer )]"
+
+ wc_magic_next_trait_level_is_1_perk_amount:0 "[EmptyScope.ScriptValue('wc_perks_needed_for_level_1_magic_trait_value')|0]"
+ wc_magic_next_trait_level_is_2_perk_amount:0 "[EmptyScope.ScriptValue('wc_perks_needed_for_level_2_magic_trait_value')|0]"
+ wc_magic_next_trait_level_is_3_perk_amount:0 "[EmptyScope.ScriptValue('wc_perks_needed_for_level_3_magic_trait_value')|0]"
+ wc_magic_next_trait_level_is_4_perk_amount:0 "[EmptyScope.ScriptValue('wc_perks_needed_for_level_4_magic_trait_value')|0]"
+ wc_magic_next_trait_level_is_last_perk_amount:0 "[EmptyScope.ScriptValue('wc_perks_needed_for_level_5_magic_trait_value')|0]"
```

Comment:
There's no text, only value so you could make it without using one script value. Do you want to remake it?

Reply:
Done